app/views/AdminView/components/Users/components/AddEditUserDialog/index.tsx

```diff
@@ -442,7 +448,15 @@ const AddEditUserDialog = ({
           </div>
         )}
       </DialogContent>
-      <DialogActions className="edit-dialog__actions">
+      <DialogActions className="add-user__actions">
```

Comment:
What's the rationale putting this on the actions instead of the actual form?

Reply:
it was solely for layout reasons but yea I can move it